src/lib/viewers/BaseViewer.js

```diff
@@ -661,16 +657,14 @@ class BaseViewer extends EventEmitter {
             return;
         }
 
-        /* global BoxAnnotations */
         const boxAnnotations = new BoxAnnotations();
```

Comment:
IMO You might want a try/catch around the instancing line. If BoxAnnotations doesn't exist, it'll blow up, potentially stopping preview from working

Reply:
👍